doc/psidk/src/development/front-ends/bundling.md

```diff
@@ -0,0 +1,21 @@
+# Bundling
+
+## Security concerns
+
+UI code is more powerful in psibase apps than it is in traditional web-apps because of its ability to make calls into [app interfaces](../../specifications/app-architecture/app-interfaces.md). If your application has its own service and app interface, your UI code can call nto your app interface, which can call into your service, which can change your application state without any explicit user confirmation.
+
+Therefore, there exists the following attack vector for psibase apps: an npm package owner could maliciously update their package after it has been bundled in with psibase app front-ends, causing it to silently call into services as the user and cause chaos.
+
+Some possible ways to avoid this risk:
```

Comment:
Hmm, that's a good analogy. But there's definitely an increased danger that's hard to articulate. 
Perhaps the same probability of risk exists, but in web3 context the impact is potentially worse due to the interconnectivity of everything. And generally in web3 the authenticator won't blindly accept session keys/bearer tokens as authentication, so in other web3 ecosystems it's less of a problem. Whereas here it *is* more like web2, and that makes it dangerous because the state on the server is more valuable than web2 state. Something like that.